addons/mail/static/src/components/autocomplete_input/autocomplete_input.js

```diff
@@ -43,6 +46,9 @@ export class AutocompleteInput extends Component {
     }
 
     _willUnmount() {
+        if (!this.root.el) {
+            return;
+        }
```

Comment:
Not a big deal because in the same situation before this PR it would just crash instead, but just to note it:

The autocomplete destroy will never be done if the last patched rendered the component empty before the unmount.

More general issue: we should never use unmount but "will update" (that we don't even have) instead